docs/src/main/sphinx/release/release-458.md

```diff
@@ -0,0 +1,114 @@
+# Release 458 (17 Sep 2024)
+
+## General
+
+* Improve performance for queries with a redundant `DISTINCT` clause. ({issue}`23087`)
+
+ ## JDBC
+
+* Add support for monitoring with OpenTelemetry. ({issue}`23458`)
```

Comment:
fyi @wendigo

Reply:
I'd phrase that differently. Add support for tracing JDBC driver with OpenTelemetry

Reply:
Or: Add OpenTelemetry tracing to JDBC driver